src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+                System.out.println("Got it. I've added this task:\n" + "  " +
+                        taskArray.get(counter));
+                counter++;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+            }
+            else if (str.equals("deadline")) {
+                String input = sc.nextLine();
+                checkInput(input);
+                String by = sc.nextLine();
+                checkInput(by);
+                taskArray.add(new Deadline(input, by));
+                System.out.println("Got it. I've added this task:\n" + "  " +
+                        taskArray.get(counter));
+                counter++;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+            }
+            else if (str.equals("event")) {
+                String input = sc.nextLine();
+                checkInput(input);
+                String period = sc.nextLine();
+                checkInput(period);
+                taskArray.add(new Events(input, period));
+                System.out.println("Got it. I've added this task:\n" + "  " +
+                        taskArray.get(counter));
+                counter++;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+            }
+            else if (str.equals("delete")) {
+                int num = sc.nextInt() - 1;
+                Task temp = taskArray.get(num);
+                taskArray.remove(num);
+                System.out.println("Noted. I've removed this task:\n" + "  " +
+                        temp.toString());
+                counter--;
+                System.out.println("Now you have " + counter + " tasks in the list.");
+                str = sc.nextLine();
+
+            }
+
+            else {
+                taskArray.add(new Task(str));
+                counter++;
+                System.out.println("added: " + str);
+                str = sc.nextLine();
+            }
+
+
+
+        }
+        if (str.equals("bye")) {
+            System.out.println("Bye. Hope to see you again soon");
+        } else {
+            taskArray.add(new Task(str));
+            counter++;
+            str = sc.nextLine();
+        }
+
+
+
+
+    }
+
+    public void returnArrayStrings(ArrayList<Task> tskArray){
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < tskArray.size(); i++){
+            if (tskArray.get(i) != null) {
+                System.out.println(Integer.toString(i + 1) + "." + tskArray.get(i));
+            } else {
+                return;
+            }
+        }
+    }
+
+    public void checkInput(String str) throws DukeException {
+        if (str.equals("")) {
+            throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+        }
```

Comment:
Try to stick to K&R style bracketing - else if should be on the same line as the closing } of the previous if clause.
Moreover, since each result is a thrown exception, there is no need for 'else'.
Also, what is 'blah' here?